tracex/extraction/logic/orchestrator.py

```diff
@@ -102,21 +102,31 @@ def initialize_modules(self):
         print("Initialization of modules successful.")
         return modules
 
-    def run(self):
+    def run(self, view=None):
         """Run the modules."""
         modules = self.initialize_modules()
+        modules_number = len(modules) + 2
```

Comment:
There should be no hardcoding here. Why is this necessary right now?

Reply:
Sure, I know what the number means. It is just that when were are adding new modules in the future we'll have to think about this hard coded number or we'll be wondering and debugging why the progress bar is not displaying correctly

Reply:
@FR-SON Is it possible that the other two modules are being executed part of the pipeline?

Reply:
We could use `self.configuration.modules`. Every Module that is used in the orchestrator should be listed there.

Reply:
sounds good, thx!